src/main/java/com/napier/sem/App.java

```diff
@@ -22,14 +22,25 @@ public static void main(String[] args)
         // Run the test query
         // dao.testQuery();
 
+        // Use-case 1.1
+        // produce a report on all countries in the world organised by largest population to smallest
+        // ArrayList<Country> countries = dao.allCountries();
+
+        // Use-cases 1.2 and 1.3
+        // produce a report on all countries in the world organised by largest population to smallest
+        ArrayList<Country> countries = dao.allCountries("region", "Southern and Central Asia");
```

Comment:
It would be good to get the constants in before we merge. It'd look something like:
`final String REGION = "region";` at the top of the file or as a property of App. Constants are all caps by convention. And the method call would look like `dao.allCountries(REGION, "Southern and Central Asia");`